### ISSUE TITLE
 False positive in no-duplicate-selector

### ISSUE THREAD

**Issue by ssivanatarajan** [stylelint/stylelint#4113]
> Clearly describe the bug
  
`Stylelint marks all the selectors as duplicated. Actually, one selector is duplicate`

> Which rule, if any, is the bug related to?

`no-duplicate-selector`

> What CSS is needed to reproduce the bug?

```css
.a,.a,.b,.c,.d{
color:red;
}
```

> What stylelint configuration is needed to reproduce the bug?

```json
{
  "rules": {
    "no-duplicate-selector": "always"
  }
}
```

> Which version of stylelint are you using?

 `10.0.1`

> How are you running stylelint: CLI, PostCSS plugin, Node.js API?

`Node API`

> Does the bug relate to non-standard syntax (e.g. SCSS, Less etc.)?

`NO`
> What did you expect to happen?

`Only one warning should be reported.`

```
Unexpected duplicate selector ".a", first used at line 2
```

> What actually happened (e.g. what warnings or errors did you get)?

```
Unexpected duplicate selector ".a", first used at line 2 (no-duplicate-selectors)
Unexpected duplicate selector ".b", first used at line 2 (no-duplicate-selectors)
Unexpected duplicate selector ".c", first used at line 2 (no-duplicate-selectors)
Unexpected duplicate selector ".d", first used at line 2 (no-duplicate-selectors)
```

**Comment by hudochenkov** [comment]
Thanks for the report and for using the template.

I can reproduce it.

**Comment by vankop** [comment]
I take this
